### PR TITLE
docs(ee/property-table) fix table formatting

### PR DESCRIPTION
**Pull request by h3xar0n**
### Summary

Formats the table for the `admin_gui_auth` reference. Instead of making a markdown table, the format is as a code block to be consistent with other property reference documentation.

### Full changelog

* Format table as a code block

### Checklist:
- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [x] Documentation
- [x] Spellchecked my updates
- [x] Ready to be merged

### CODE REVIEW

**Comment by coopr** [comment]
This doesn't make sense to me as a table, either table-formatted or code-formatted. 

I'd suggest simplifying and clarifying by just presenting the two options code-formatted, and linked to their respective plugin pages:

`[basic-auth](/hub/kong-inc/basic-auth/)` or `[ldap-auth-advanced](/hub/kong-inc/ldap-auth-advanced/)`

Ideally these URLs ^^ would be "version locked" to this version, but I versioned plugin docs either don't exist yet, or can't be referred to via URL, so... I guess that could get fixed later.

**Comment by travispamaral** [comment]
I agree that the table is a little confusing (value to use / Authentication Type). Maybe reword or just show the format needed for conf file

**Comment by kinman** [comment]
oops, didn't finish my thought. Thinking out loud on a PR, we might want to distinguish more clearly between authentication for Kong Manager and authentication for Kong-proxied services.
